client/src/components/Markdown/Markdown.vue

```diff
@@ -1,3 +1,86 @@
+<script setup lang="ts">
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faDownload, faEdit } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { computed, onMounted, ref, watch } from "vue";
+
+import { parseMarkdown } from "./parse";
+
+import MarkdownDefault from "./Sections/MarkdownDefault.vue";
+import MarkdownGalaxy from "./Sections/MarkdownGalaxy.vue";
+import LoadingSpan from "@/components/LoadingSpan.vue";
+import StsDownloadButton from "@/components/StsDownloadButton.vue";
+
+library.add(faDownload, faEdit);
```

Comment:
Adding icons to the library is no longer necessary as long as you don't use the "string reference" of the icon (which is not recommended anyway).
So the way we reference icons now is like this: `<FontAwesomeIcon :icon="faEdit" />` This is cleaner, more "searchable," and correctly includes the icon in the bundle if needed.